server/testdb/test_run_event.go

```diff
@@ -136,10 +135,6 @@ func readTestRunEventFromRows(rows *sql.Rows) (model.TestRunEvent, error) {
 	)
 
 	if err != nil {
-		if errors.Is(err, sql.ErrNoRows) {
```

Comment:
the controller compares with `errors.Is(err, sql.ErrNoRows)` now. this func checks the wrapped errors, so it will correctly assert that the error is `sql.ErrNoRows` without the need to return a custom error